### PR TITLE
Add community page with dissemination material

### PR DESCRIPTION
**Pull request by ktsakalozos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,6 +244,10 @@ To speed-up a build you can reuse the binaries already downloaded from a previou
 snap install microk8s_v1.10.3_amd64.snap --classic --dangerous
 ```
 
+## How to Use MicroK8s
```
Comment (tvansteenburgh) [inline README.md]: I'd change this to "Who's Using MicroK8s"

File: docs/community.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# MicroK8s In The Wild
+
+Welcome to the spot MicroK8s In The Wild page. Here you can find, events, tutorials, blog posts and talks on MicroK8s.
```
Comment (tvansteenburgh) [inline docs/community.md]: s/spot//

Also remove the comma after "find".

File: README.md (hunk 1 above)
Comment (ktsakalozos) [reply README.md]: Done

File: docs/community.md (hunk 2 above)
Comment (ktsakalozos) [reply docs/community.md]: Done

**Comment by ktsakalozos** [comment]
Merging this, thank you.

**Pull request merged**